### PR TITLE
Issue 1621 - reorder Project Team

### PR DESCRIPTION
**Pull request by acrymble**
As per issue #1621 this is a test to see if I've been able to reorder the Project Team into publication-based teams. I've had to add a new role for people with ProgHist ltd roles, which lets them cluster together.

If this works and we like it, I will have to get the proghist role description in roles.md translated.

This shouldn't be merged unless we agree we like it.

Closes issue #1621 

### Checklist

- [x] Assign yourself in the "Assignees" menu
- [x] Assign at least one individual or team to "Reviewers"
  - [x] if the text needs to be translated, assign the relevant language team(s) as "Reviewers" and tag both the team as well as the managing edtor in your PR. Please follow the [translation request guidelines](https://github.com/programminghistorian/jekyll/wiki/Requesting-Translation-Guidelines) when writing your PR description
- [x] Add the appropriate "Label"
- [x] [Ensure the Travis CI checks pass](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#4-check-your-pr-status)
- [x] [Check the live preview of your PR on Netlify](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#5-preview-how-your-pr-looks-when-built-into-html)
- [x] If this PR closes an open issue, add the phrase `Closes #ISSUENUMBER` to the description above

*If you are having difficulty fixing Travis errors, first consult <https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions> carefully. Then contact the technical team if you need further help.*

### CODE REVIEW

**Comment by mdlincoln** [comment]
reminder Adam to please use the built-in PR checklist instead of deleting it

File: _includes/project-team-loop.html [hunk 1]
```diff
@@ -7,14 +8,60 @@
...
 
-{% assign teammembers = site.data.ph_authors | sort: "sortname" %}
-{% for member in teammembers %}
-  {% if member.team == true %}
-    {% capture membername %}{{ member.name }}{% endcapture %}
-    {% include contact-info.html name=membername %}
-  {% endif %}
+
+<h3>Programming Historian in English</h3> 
+{% assign englishmembers = site.data.ph_authors | sort: "sortname" %}
```
Comment (mdlincoln) [inline _includes/project-team-loop.html]: This looks good. 

You can rewrite each of the blocks to filter the team members more directly like so:

```
{% assign englishmembers = site.data.ph_authors | where_exp: "member", "member.team_roles contains 'editorial'" | where_exp: "member", "member.team == true" | sort: "sortname" %}
{% for member in englishmembers %}
{% include contact-info.html name=member.name %}
{% endfor %}
```

This filters twice: first, checking the `team_roles` contains the right team, and then next filtering to check that `team` is `true`.

Also, note that you don't need to use the `capture` tag - you can directly pass `member.name` to the `contact-info.html` include.

Doing this reduces the number of `if` and `for` statements so it's easier to see what the code is doing.

**Review by mdlincoln — CHANGES_REQUESTED** [review]
This structure looks almost perfect @acrymble 👏  The code change I'm requesting refactors it so that it's clearer which filters are being applied.

File: _includes/project-team-loop.html (hunk 1 above)
Comment (acrymble) [reply _includes/project-team-loop.html]: Thanks. Are these changes I have to add, or have you added them? Sorry, still learning the technical side of things on github.

Comment (mdlincoln) [reply _includes/project-team-loop.html]: You need to add them. You can use the code I put in this comment, but you need to add it for each of the teams, adjusting it appropriately each time.

Comment (mdlincoln) [reply _includes/project-team-loop.html]: Interestingly, looking at the ToC, I think that you will need to express the `<h3>` header using Markdown in order to get it to be recognized by JEkyll. So replace

```html
<h3>Programming Historian in English</h3>
```

with

```md
### Programming Historian in English
```

**Review by mdlincoln — DISMISSED** [review]
The code changes look great. Now I think this just needs the ES and FR translations?

**Comment by acrymble** [comment]
Thanks @mdlincoln.

@programminghistorian/french-team and @programminghistorian/spanish-team could I please have translations of the following text:

> This member holds a role in the not-for-profit company ProgHist Ltd, which supports the publications.

**Comment by fdlaramee** [comment]
@acrymble : Ce(tte) membre occupe un rôle au sein de la compagnie sans but lucratif ProgHist Ltd, qui soutient les publications.

**Comment by jenniferisasi** [comment]
En español: 

Este miembro funge un cargo en la compañía sin ánimo de lucro ProgHist Ltd, que gestiona las publicaciones.

**Review by mdlincoln — APPROVED** [review]

**Comment by acrymble** [comment]
I think this is an improvement and I'd like to merge. Anyone want to express views? https://deploy-preview-1637--ph-preview.netlify.com/en/project-team

**Comment by jenniferisasi** [comment]
> Anyone want to express views?

I saw it this morning and I really like it. Thanks for the work you put into it.

**Review by rivaquiroga — APPROVED** [review]

**Comment by rivaquiroga** [comment]
Looks great! Thank you

**Comment by acrymble** [comment]
@rivaquiroga @jenniferisasi thanks. And @mdlincoln thanks for the technical support. I haven't heard any negative comments going through the process, so I'll merge this in. If anyone wants anything tinkered with in future, we can address it (no guarantee I got it perfect yet).

**Pull request merged**